webapp/src/constants.ts

```diff
@@ -12,6 +12,7 @@ export const CALL_RECORDING_POST_TYPE = 'custom_calls_recording';
 export const CALL_TRANSCRIPTION_POST_TYPE = 'custom_calls_transcription';
 export const LIVE_CAPTION_TIMEOUT = 5000;
 export const HOST_CONTROL_NOTICE_TIMEOUT = 5000;
+export const DEGRADED_CALL_QUALITY_ALERT_WAIT = 20000;
```

Comment:
I think we should make this dynamic based on the derivative of the mos change. That way if the connection goes really bad we can show it again without waiting.

&nbsp;
&nbsp;
&nbsp;
&nbsp;
 
I am just kidding :)

Reply:
😭